relay-server/src/metrics_extraction/event.rs

```diff
@@ -1119,7 +1119,8 @@ mod tests {
         project.sanitize();
 
         let config = project.metric_extraction.ok().unwrap();
-        let metrics = extract_metrics(event.value().unwrap(), false, &config, 200, None);
+        let combined_config = (&config).into();
```

Comment:
Not sure if that's better:

```suggestion
        let combined_config = CombinedMetricExtractionConfig::from(&config);
```

Reply:
I'll leave this for the next PR, where I refactored these tests a bit: https://github.com/getsentry/relay/pull/3504